Dungeoneer/src/com/interrupt/dungeoneer/collision/CollisionTriangle.java

```diff
@@ -0,0 +1,20 @@
+package com.interrupt.dungeoneer.collision;
+
+import com.badlogic.gdx.math.Vector3;
+import com.badlogic.gdx.utils.Triangle;
+
+public class CollisionTriangle extends Triangle {
+
+    public enum TriangleCollisionType { WORLD, WATER };
```

Comment:
Would `LAVA` also need to be an option here? I would think that decals aren't allowed/make sense here either.

Reply:
Tested. Works fine with lava as well!